README.md

```diff
@@ -34,18 +34,28 @@
 To install from PyPI and run:
 
 ```bash
-$ pip install -U pip
-$ pip install titiler.pgstac
+# Make sure to have pip up to date
+$ python -m pip install -U pip
+
+# Install psycopg2 or psycopg2-binary
+$ python -m pip install psycopg2-binary  # or psycopg2
+
+$ python -m pip install titiler.pgstac
 ```
 
 To install from sources and run for development:
 
 ```
 $ git clone https://github.com/stac-utils/titiler-pgstac.git
 $ cd titiler-pgstac
-$ pip install -e .
+$ python -m pip install -e .
 ```
 
+### `psycopg2` requirement
+
+`titiler.pgstac` depends on the `psycopg2` library. Because there are two ways of installing this package (`psycopg2` or `psycopg2-binary`), it is let to the user to install it. if you should to install `psycopg2`, you'll need to have Postgres installed.
```

Comment:
```suggestion
`titiler.pgstac` depends on the `psycopg2` library. Because there are two ways of installing this package (`psycopg2` or `psycopg2-binary`), the user must install this separately from `titiler.pgstac`. `psycopg2-binary` is a binary wheel distribution of the `psycopg2` package and is simpler for development. `psycopg2` is [generally recommended](https://github.com/psycopg/psycopg2/blob/e7c5f95bf60f4d243733e3cbcedb365d76aa28d5/README.rst#installation) for production use. Note that to install `psycopg2`, you'll need to have Postgres headers installed and available for the compilation process.
```